projects/mysadna/src/app/shared/contributor/contributor.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+import { User } from '@/proto';
+
+@Component({
+  selector: 'app-contributor',
```

Comment:
Instead of create same code several times, we can create shared component with the code